src/ethereum/fees/feeProviders.ts

```diff
@@ -65,10 +67,10 @@ export const fetchFeesFromEvmScan = async (
   fetch: EdgeFetchFunction,
   currencyInfo: EdgeCurrencyInfo,
   initOptions: EthereumInitOptions,
-  log: EdgeLog
+  log: EdgeLog,
+  networkInfo: EthereumNetworkInfo
```

Comment:
I noticed this the other day, and I didn't say anything, but maybe it's worth mentioning anyhow: `PluginEnvironment<NetworkInfo>` has all these things inside. You get `currencyInfo`, `networkInfo`, `log`, `initOptions`, and while it doesn't have `fetch` itself, you do get `io`.

However, the reason I didn't say anything is because this would be a bit bigger refactor, and I don't necessarily want to spend time on it, given the deadline. Just thoughts for later.

Reply:
Definitely worth mentioning and that would be a great commit in a future "Cleanup Ethereum" PR